scripts/buildTranscendJsonSchema.ts

```diff
@@ -5,7 +5,7 @@
  * Most IDEs will thus autodetect `transcend.yml` and apply linting/autocomplete/intellisense.
  *
  * ... or, if the YAML file is differently named, users can add this comment to the top of the YAML file:
- * `# yaml-language-server: $schema=https://raw.githubusercontent.com/transcend-io/cli/main/transcend-policy-yml-schema.json`
+ * `# yaml-language-server: $schema=https://raw.githubusercontent.com/transcend-io/cli/main/transcend-yml-schema-latest.json`
```

Comment:
does this remove the versions? they seem somewhat valuable to maintain. 

im ok to remove but if removed, should we delete the old ones?

Reply:
It doesn't remove them. We should keep them so folks can still pin like this: https://github.com/transcend-io/cli/pull/367/files#r1861056500

Reply:
@bencmbrook should you add back the logic to generate those files then? maybe i missed it, but it seems like you removed the -v6.json generation here

Reply:
https://github.com/transcend-io/cli/pull/367/files/df0cfcc8be6f88e6b6c5210b65259ed0ffabaf77#r1861078035